src/lib/flatten/tsc.ts

```diff
@@ -17,6 +17,7 @@ export async function downlevelWithTsc(inputFile: string, outputFile: string): P
     target: ScriptTarget.ES5,
     module: ModuleKind.ES2015,
     allowJs: true,
+    importHelpers: true,
```

Comment:
Didn't quite get what you mean, you need to importHelpers while downleveling as without it it won't use `tslib`

Reply:
Is the change affecting just the two code occurences?

1. https://github.com/dherges/ng-packagr/pull/428#discussion_r165038381
2. https://github.com/dherges/ng-packagr/pull/428#discussion_r165038237

Reply:
And https://github.com/dherges/ng-packagr/pull/428/files#diff-a4eee2c0b9136c939507b9942ac400a9R30

https://github.com/dherges/ng-packagr/pull/428/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R56

and added some integration tests.